src/System.Runtime.InteropServices.RuntimeInformation/tests/project.json

```diff
@@ -12,7 +12,8 @@
       "exclude": "compile"
     },
     "Microsoft.xunit.netcore.extensions": "1.0.0-prerelease-00508-01",
-    "Microsoft.DotNet.BuildTools.TestSuite": "1.0.0-prerelease-00520-02"
+    "Microsoft.DotNet.BuildTools.TestSuite": "1.0.0-prerelease-00520-02",
+    "System.Diagnostics.FileVersionInfo": "4.0.0"
```

Comment:
The dependency closure of `System.Diagnostics.FileVersionInfo` is huge on non-Windows. It seems overkill to pull it in just to get a file version.